src/app/core/footer/footer.component.css

```diff
@@ -4,6 +4,7 @@ footer {
   margin-bottom: 0;
   width: 100%;
   background-color: var(--darkest-accent);
+  font-size: 0.8em;
```

Comment:
This font size is a bit too small to be readable.

Reply:
Facebook uses **12px** and with 0.8em I'm seeing **13px** on my screen. I think the text is fine.